var/spack/repos/builtin/packages/heaptrack/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Heaptrack(CMakePackage):
+    """Heaptrack is a heap memory profiler that traces all memory allocations
+    and annotates these events with stack traces.
+    """
+
+    homepage = "https://github.com/KDE/heaptrack"
+    url      = "https://github.com/KDE/heaptrack/archive/v1.1.0.tar.gz"
+
+    version('1.1.0', sha256='bd247ac67d1ecf023ec7e2a2888764bfc03e2f8b24876928ca6aa0cdb3a07309')
+
+    depends_on('boost@1.41:')
+    depends_on('cmake@2.8.9:')
+    depends_on('elfutils')
+    depends_on('libunwind')
+    depends_on('zlib')
+    depends_on('zstd')
+
+    variant('build_type', default='Release',
+            description='CMake build type',
+            values=['Release'])
```

Comment:
Why only `Release` as a build type?

Reply:
I originally added this as the [official build recommends](https://github.com/KDE/heaptrack#compiling) recommends Release.

Nevertheless, the other builds seem to work so I removed this.

Reply:
As far as I'm concerned you can definitely override to use `Release` as a default if it seems appropriate to you. What I didn't understand is why you allowed **only** that value. In any case fine with me if you want to leave the package like this.

Reply:
Allowing multiple values requires testing if the software builds with these values and as there was no indication in the repo that the software would build/could be built with other build_types, I considered it safer to limit the build to whatever was recommended by the developers.

I tested Spack's default build options and they seemed to work fine (no idea if the end product is any different, though).